### PR TITLE
Updated version of Google gemini-1.5-pro

### PR DESCRIPTION
**Pull request by xiaozhou26**

### CODE REVIEW

**Comment by H0llyW00dzZ** [comment]
it still not available yet

image

**Comment by fred-bf** [comment]
@xiaozhou26 have you got the access to gemini-pro 1.5 api? I cannot validate the api before Google Published in ai studio

File: app/constant.ts [hunk 1]
```diff
@@ -87,7 +87,7 @@ export const Azure = {
 
 export const Google = {
   ExampleEndpoint: "https://generativelanguage.googleapis.com/",
-  ChatPath: "v1beta/models/gemini-pro:generateContent",
+  ChatPath: "v1beta/models/gemini-pro:generateContent",
```
Comment (fred-bf) [inline app/constant.ts:90]: Gemini's model is dynamically spliced to the path, and it needs to be modified to be dynamic

**Review by fred-bf — CHANGES_REQUESTED** [review]

File: app/constant.ts (hunk 1 above)
Comment (mountainguan) [reply app/constant.ts:90]: > Gemini's model is dynamically spliced to the path, and it needs to be modified to be dynamic

so, if I want to use gemini-1.5-pro，i should change the _ChatPath_ as “v1beta/models/models/gemini-1.5-pro:generateContent”?

**Comment by jeus1609** [comment]
Excuse me, is it available now?

**Pull request closed without merging**